content/docs/community/component-guide.md

```diff
@@ -0,0 +1,668 @@
+
+Adjust the faucet to your preferred temperature and flow rate.
+
+## Fill and Drink
+
+Fill the glass to desired level and enjoy your water.
+
+</Steps>
+
+---
+
+## Tabbed Content
+
+Components for organizing content into tabs.
+
+### CodeTabs
+
+Multi-language code examples with tabs.
+
+````mdx
+<CodeTabs labels={["JavaScript", "Python", "Go"]}>
+
+```javascript
+const { Client } = require('pg');
+const client = new Client({
+  connectionString: process.env.DATABASE_URL,
+});
+await client.connect();
+```
+
+```python
+import psycopg2
+import os
+
+conn = psycopg2.connect(os.environ["DATABASE_URL"])
+cur = conn.cursor()
+```
+
+```go
+import (
+    "database/sql"
+    _ "github.com/lib/pq"
+)
+
+db, err := sql.Open("postgres", os.Getenv("DATABASE_URL"))
+```
+
+</CodeTabs>
+````
+
+**Live Preview:**
+
+<CodeTabs labels={["JavaScript", "Python", "Go"]}>
+
+```javascript
+const { Client } = require('pg');
+const client = new Client({
+  connectionString: process.env.DATABASE_URL,
+});
+await client.connect();
+```
+
+```python
+import psycopg2
+import os
+
+conn = psycopg2.connect(os.environ["DATABASE_URL"])
+cur = conn.cursor()
+```
+
+```go
+import (
+    "database/sql"
+    _ "github.com/lib/pq"
+)
+
+db, err := sql.Open("postgres", os.Getenv("DATABASE_URL"))
+```
+
+</CodeTabs>
+
+---
+
+### Tabs
+
+General tabbed content (not just code). For code-specific tabs, use [CodeTabs](#codetabs) instead.
+
+````mdx
+<Tabs labels={["Console", "CLI", "API"]}>
+<TabItem>
+Create a database using the Neon Console by navigating to your project dashboard and clicking "Create Database".
+</TabItem>
+<TabItem>
+Use the Neon CLI to create a database:
+
+```bash
+neon databases create --name my-database
+```
+
+</TabItem>
+<TabItem>
+Use the API to create a database:
+
+```bash
+curl -X POST https://console.neon.tech/api/v2/projects/my-project/databases \
+  -H "Authorization: Bearer $NEON_API_KEY"
+```
+
+</TabItem>
+</Tabs>
+````
+
+**Live Preview:**
+
+<Tabs labels={["Console", "CLI", "API"]}>
+<TabItem>
+Create a database using the Neon Console by navigating to your project dashboard and clicking "Create Database".
+</TabItem>
+<TabItem>
+Use the Neon CLI to create a database:
+
+```bash
+neon databases create --name my-database
+```
+
+</TabItem>
+<TabItem>
+Use the API to create a database:
+
+```bash
+curl -X POST https://console.neon.tech/api/v2/projects/my-project/databases \
+  -H "Authorization: Bearer $NEON_API_KEY"
+```
+
+</TabItem>
+</Tabs>
+
+---
+
+## Content Organization
+
+Components for structuring and organizing page content.
+
+### TechCards / DetailIconCards
+
+Technology cards with icons, titles, and descriptions. These components use different [icon systems](/docs/community/component-icon-guide) - see the [comparison table](#techcards-vs-detailiconcards-vs-docslist) below to choose the right one.
+
+#### TechCards
+
+Standard technology cards layout using [TechCards icons](/docs/community/component-icon-guide#techcards-icons):
+
```

Comment:
Any way we can fix the line breaks?

Reply:
It appears not as the Prettier job is reverting the changes!